src/nimble.nim

```diff
@@ -802,6 +802,15 @@ This should ideally be a valid SPDX identifier. See https://spdx.org/licenses/.
 Please specify a valid SPDX identifier.""",
       "MIT"
     )
+  var pkgBackend = options.promptList(
+    """Package Backend?
+c    - Compile to binary.
+cc   - Transpile to C.
```

Comment:
Change these to "Compile using the C backend" (transpilation is a dirty word)

Also, remove the first option and rename ``cc`` to just `c`.

Reply:
Will do! Mind if I ask why it's a dirty word? :P

Reply:
Does gcc transpile to asm? :)

Reply:
I guess not :)

Reply:
Since `c`/`cc` are semantically different (one means "use default backend" and the other "use the c backend") it might make sense to offer both. The default backend could theoretically be changed to `cpp` in the future.

Reply:
Whoops, resolved this without seeing your comment. That's a good point, maybe we should consider that.

Reply:
We can solve that when the time comes :)